src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 33px 19px 16px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__h1 {
+    margin: 0;
+    margin-bottom: 4px;
+    max-width: 166px;
+    color: var(--color-for-main-text);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__text-code {
+    margin: 0;
+    margin-bottom: 16px;
+    max-width: 98px;
+    color: var(--color-for-secondary-text);
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__div-for-stars {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-p {
+      margin: 0;
+      text-align: right;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__div-price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-h2 {
+      margin: 0;
+      color: var(--color-for-secondary-text);
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-p {
+      margin: 0;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. it's better to use padding